app-text/alanvar/alanvar-0.5.0.recipe

```diff
@@ -0,0 +1,91 @@
+SUMMARY="Al-Anvar, a Quran Study Software "
```

Comment:
```diff
-SUMMARY="Al-Anvar, a Quran Study Software "
+SUMMARY="A Quran Study Software"
```
That's because:
* `SUMMARY` does not need to mention the name of the app.
* The white space before the closing double quotes is useless.